### ISSUE TITLE
create-figma-plugin: Could not resolve "@create-figma-plugin/utilities"

### ISSUE THREAD

**Issue by danielamram** [yuanqing/create-figma-plugin#132]
Hey there, 
After a successful `npx create-figma-plugin` cli command tried to `npm watch` and i'm experiencing issues with esbuild:
```shell
error esbuild error
    Build failed with 1 error:
    src/main.ts:1:29: ERROR: Could not resolve "@create-figma-plugin/utilities"
✖  Error building interactive interface
```

The package exists in `node_modules` folder so I'v tried to manage these workarounds with no luck:
- delete `package-lock.json` & `node_modules` and rerun `npm install`
- different templates installations `plugin/preact-rectangles` | `plugin/preact-resizable` | `plugin/react-editor`
- install `@create-figma-plugin/utilities` package manually
- different node versions 14|16|18

Any ideas how to manage? i'm not familiar with esbuild, I can dive into the build script if needed but just wanted to make sure i'm not missing anything.

Cheers and thanks for the great work there ! 🍻

**Comment by danielamram** [comment]
It does seems like the issue came up on version `@create-figma-plugin/build@2.13`, I tried a fixed `2.1.2` version and it works well, the only relevant change between `2.1.2` -> `2.1.3` is [this commit](https://github.com/yuanqing/create-figma-plugin/commit/efba7a2baca3000e2963f4156b40cb9f8654995f) WDYT

**Comment by yuanqing** [comment]
@danielamram – I’m not able to reproduce this issue, unfortunately. Does building work if you use `2.1.5` for all packages?

```sh
$ node --version
v16.16.0
$ npx --yes -- create-figma-plugin@2.1.5 --template plugin/preact-rectangles
info Copying "plugin/preact-rectangles" template...
info Resolving package versions...
info Installing dependencies...
success Done

First:
  cd preact-rectangles

To build the plugin:
  npm run build
  
To watch for code changes and rebuild the plugin automatically:
  npm run watch

$ cd preact-rectangles
$ npm run build

> build
> build-figma-plugin --typecheck --minify

info Type checking...
success Type checked in 0.945s
info Building...
success Built in 0.352s
```
